packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/corellium/api_test.py

```
@@ -0,0 +1,215 @@
+    project = Project('d59db33d-27bd-4b22-878d-49e4758a648e', 'Default Project')
+    device = Device(name='rd1ae', type='automotive', flavor='kronos',
+                    description='', model='kronos', peripherals=False, quotas=dict())
+    instance = api.create_instance('my-instance', project, device, '1.0')
+
+    assert instance is not None
+    assert instance.id
+
+
+@pytest.mark.parametrize(
+    'status_code,data,msg',
+    [ 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)